cql3/statements/alter_table_statement.cc

```diff
@@ -341,6 +341,10 @@ std::pair<schema_builder, std::vector<view_ptr>> alter_table_statement::prepare_
                 }
             }
 
+            if (_properties->get_synchronous_updates_flag()) {
```

Comment:
As I noted in https://github.com/scylladb/scylla/issues/10545#issuecomment-1123920515, it would be nice to try to understand and document somewhere (preferably not a commit message that nobody will ever see again. Maybe docs/?) exactly what are the real advantages and disadvantages of this option. We discussed on that thread some non-options.

That being said, the **feature** can be committed even before we have good documentation for it.

Reply:
> Are you going to change it as well?

I will, definitely, it's the same kind of oversimplification; thanks